ehitchhiking-ui/src/app/shared/modals/profile-modal/profile-modal.component.sass

```diff
@@ -0,0 +1,83 @@
+@import '../../../../styles.sass'
+
+.profile-container
+    position: relative
+    font-family: $app-font
+    background-color: mat-color($primary, A100)
+    overflow: hidden
+    height: auto
+    width: 40rem
+
+    .profile-header
+        width: 100%
+        height: 4rem
+        font-size: $headline
+        @include flexbox()
+        background-color: mat-color($accent, 700)
+        color: $white
+        
+    .profile-data
+        @include padding(0, $default-offset, 0, $default-offset)
+
+        .profile-user-info
+            position: relative
+            @include flexbox(space-around)
+            font-size: $subheading-2
+
+            .profile-user-photo
+                flex-grow: 1
+                padding-right: $default-offset
+                object-fit: cover
+            .material-icons
+                margin-left: $default-offset*2
+                font-size: $display-1
+            .profile-user-contacts
+                @include flexbox(space-around, center, column)
+                flex-grow: 2
+        hr
+            color: mat-color($primary, A200)
+
+        .profile-car-info
+            &:first-child
+                margin-top: $default-offset*2
+            &:last-child
+                margin-bottom: $default-offset
+            &-desc
+                @include flexbox(space-around, center, row)
+                .car-info-delete:hover
+                    color: mat-color($warn)
+                &-item
+                    width: 25%
+                    text-align: center
+            &-title 
+                width: 10%
+                @include flexbox(flex-start)         
+                
+        .profile-add-car
+            position: absolute
+            top: $default-offset * 12.8
```

Comment:
you should not compute top, right property using $default-offset

Reply:
done